packages/react-bindings/src/styles/calculateAnimationTimeout.ts

```diff
@@ -0,0 +1,10 @@
+import convertCssTimeToNumber from './convertCssTimeToNumber'
+import * as _ from 'lodash'
+
+const calculateAnimationTimeout = (duration: string, delay: string | undefined): number => {
+  if (_.isNil(duration) || duration === 'inherit' || duration === 'initial') return 0
+  if (!_.isNil(delay) && (delay === 'inherit' || delay === 'initial')) return 0
```

Comment:
If it's under `react-bindings` I would like to avoid dependency on Lodash

Reply:
We shouldn't have added it as dependency if we don't want it. I am already using `_.isNil`, `_.endsWith` and `_.toNumber`... If we decide to remove it from the package, we can refactor all files that use lodash